idp/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:buster as gobuild
```

Comment:
Is there a need to build from the tip of master, or could we use the `confd` from the base image? The base image one seems to work for me.  If so, we can use the simplification in https://github.com/emetsger/idc-isle-buildkit/pull/1